server/sandbox_cri.go

```diff
@@ -0,0 +1,38 @@
+package server
```

Comment:
what is the point of this file? All sandbox related cri operations?

Reply:
Hm, no I think we could put them into dedicated files like we currently have it.